install_config/topics/configuring_azure_authorization.adoc

```diff
@@ -0,0 +1,77 @@
+////
+Module included in the following assemblies:
+
+install_config/configuring_azure.adoc
+////
+
+[id='configuring-azure-permissions_{context}']
+= Configuring authorization for Microsoft Azure
+
+*Azure roles*
+
+Configuring Microsoft Azure for {product-title} requires the following role:
+
+[cols='1,3']
+|===
+
+|Contributor
+|To create and manage all types of Microsoft Azure resources.
+
```

Comment:
@bfallonf Does this one item need the table? 
_Configuring Microsoft Azure for {product-title} requires the Contributor role to create and manage all types of Microsoft Azure resources._

Reply:
This would be an Azure role

Reply:
Clarified.